pulsar-client/src/main/java/org/apache/pulsar/client/impl/MultiTopicsConsumerImpl.java

```diff
@@ -590,6 +591,10 @@ String getHandlerName() {
         internalConsumerConfig.setSubscriptionName(subscription);
         internalConsumerConfig.setConsumerName(consumerName);
         internalConsumerConfig.setMessageListener(null);
+        // output the org topic name
+        if (null != orgConf && internalConsumerConfig.getTopicNames().isEmpty()) {
+            internalConsumerConfig.getTopicNames().addAll(orgConf.getTopicNames());
```

Comment:
@xuesongxs It's not a correct fix here, we can't add all topics to the internal consumer since the internal consumer only connect to a single topic. I think we can fix the log directly?

Reply:
@xuesongxs Can you remove here? I think it will not been used for the PR?

Reply:
@codelipenghui  I have restored the transformation of multitopics consumerimpl.java, and now I only keep the transformation of consumerstatsrecordimpl.java